src/components/Header/Header.js

```diff
@@ -2,11 +2,11 @@ import React from "react";
 import logo from "./logo.png";
 import { MdShoppingCart, MdSearch } from "react-icons/md"
 
-function Header() {
+function Header(params) {
```

Comment:
It would be better if instead of only `params` you can destructure what it contains, so it's easy to know what the params really are
```suggestion
function Header( { setIsHomePage }) {
```